src/main/python/ttconv/imsc/elements.py

```diff
@@ -191,7 +199,8 @@ def from_xml(_parent_ctx: typing.Optional[TTMLElement.ParsingContext], xml_elem)
     return tt_ctx
 
   @staticmethod
-  def from_model(model_doc: model.Document) -> et.Element:
+  def from_model(model_doc: model.Document, frame_rate = Fraction(1,1)) -> et.Element:
```

Comment:
Can `frame_rate` be `None`? How would the caller specify that conversion to `f` is not desired?

Reply:
Done. None means use hh:mm:ss:ms otherwise frames